website/blog/2021-09-22-release-0-3.md

```diff
@@ -0,0 +1,152 @@
+---
+slug: release-sb-v0.3
+title: Releasing SuperBench v0.3
+author: Peng Cheng
+author_title: SuperBench Team
+author_url: https://github.com/cp5555
+author_image_url: https://github.com/cp5555.png
+tags: [superbench, announcement, release]
+---
+
+We are very happy to announce that **SuperBench 0.3.0 version** is officially released today!
+
+You can install and try superbench by following [Getting Started Tutorial](https://microsoft.github.io/superbenchmark/docs/getting-started/installation).
+
+## SuperBench 0.3.0 Release Notes
+
+### SuperBench Framework
+
+#### Runner
+
+- Implement MPI mode.
+
+#### Benchmarks
+
+- Support Docker benchmark.
+
+### Single-node Validation
```

Comment:
The following two categories look a little ambiguous:
### Single-node Validation
### AMD GPU Support

Could we merge `AMD GPU Support` into `Single-node Validation` or split `Single-node Validation` to 'NVIDIA GPU Support' and `Other Benchmarks`?

Reply:
Single-node validation is for general purpose actually. I can support both platform.

Reply:
If so, we should unify the NCCL/RCCL and Memory category? Since they appear in both `general purpose` category and `AMD platform` category.